buildSrc/build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id 'java-gradle-plugin'
+}
+
+gradlePlugin {
+    plugins {
+        'github-publish' {
+            id = 'io.radar.ghpublish'
+            implementationClass = 'io.radar.ghpublish.GitHubPublishPlugin'
+        }
```

Comment:
Do we still need this / is it being used?

Reply:
It is not being used anymore. I kept it in here in case we wanted to use it at some point

Reply:
I'd be in favor of moving it out since we're not using it right now (can stash these changes in a branch if we want to refer to them later)

Reply:
Removed